libs/cart/guides/state.md

```diff
@@ -0,0 +1,86 @@
+# State
+
+- [State](#state)
+  - [Setting up your `AppModule`](#setting-up-your-appmodule)
+  - [Using the Facade](#using-the-facade)
+
+`@daffodil/cart` provides a fully featured state library to streamline the management of your application's state as well as driver interaction.
+
+The facade is an abstraction that provides all the functionality needed for standard use. It is the recommended way to interact with the Daffodil state layer.
+
+## Setting up your `AppModule`
+
+To get started, import the `DaffCartModule` in your `AppModule`. Next, import `StoreModule.forRoot({})`, which will be relevant later on when using the redux and state management features of the cart module.
+
+```typescript
+@ngModule({
+  imports:[
+    StoreModule.forRoot({}),
+    DaffCartModule
+  ]
+})
+export class AppModule {}
+```
+
+## Using the Facade
```

Comment:
I think this should be "Using the Facade to Access State" and it could be contrasted with another heading of "Accessing State Directly Through Daffodil Selectors". I think that could be useful. What do you think? Maybe it would make more sense for that to go under the advanced guides?

Reply:
I think it makes to have some docs that talk about using the selectors, especially since we have a somewhat non-standard way of instantiating them. Since its not the recommended way to access state though, I think it should be an advanced guide or separate section that's presented as an alternative to using the facade.

I think this section header should stay as it is because the section also documents dispatching actions through the facade.

Reply:
added selector doc